actions/websocket_actions.jsx

```diff
@@ -171,6 +172,10 @@ export function reconnect(includeWebSocket = true) {
         dispatch(TeamActions.getMyTeamUnreads());
     }
 
+    if (state.websocket.lastDisconnectAt) {
```

Comment:
LGTM though i have a question about an alternate approach. @hmhealey What are your thoughts on having this as the responsibility of react component to call this on reconnect? Probably the `channel_view` component or something around that which is at the top level of channel route. 

It would be hard to call different API's without knowing the exact state of UI or we might be ending up calling API's the state does not need ATM.

Reply:
The problem is that there's not really a component that owns this sort of thing. As long as we have the websocket connected, even if we're not on the channel screen, we still want to make sure that things like this are properly updated. I think this is something that has to be done independent of the components for that reason. It might be nice if we had some specific part of the code responsible for syncing up data like this, but that's just websocket_actions.jsx for now

Reply:
@hmhealey My idea was to designate the websocket status flag to different components and let those components decide on the API calls to make when status changes from disconneted to connected..

I don't think we refresh the search results on websocket reconnect same case with different other UI components but if we pass in the socket status flag to RHS we can make the API calls on reconnect making the UI in sync with the server.

I just feel that we will have different other instance(Pinned messages, flagged messages etc) which will have state data on socket refresh we don't ask for

I am good with this as the change will be even out of scope for the PR. We can iterate on the idea in a channel somewhere